CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 3.16)
+
+# Set basic project properties
+project(RCM-f90
+    VERSION 0.1
+    DESCRIPTION "RCM f90 is a modified version of the code available at https://people.math.sc.edu/Burkardt/f_src/rcm/rcm.html to facilitate building as a library, released under an LGPL license - see LICENSE."
+    LANGUAGES Fortran
+)
+
+# Set up install directories
+include(GNUInstallDirs)
+set(INSTALL_INCLUDEDIR "${CMAKE_INSTALL_INCLUDEDIR}/rcm-f90")
+set(INSTALL_MODULEDIR "${INSTALL_INCLUDEDIR}/mod")
```

Comment:
The current Makefile uses `include/` for module files, to avoid changing users we should keep the same directory layout.